superset-frontend/src/components/ListView/ListView.tsx

```diff
@@ -28,11 +28,15 @@ import {
   Row,
   // @ts-ignore
 } from 'react-bootstrap';
+// @ts-ignore
+import { default as SelectComponent } from 'react-select';
```

Comment:
silly NIT/question - just wondering why you chose this over `import SelectComponent from 'react-select';`